### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by ricardopereira**
#637

### CODE REVIEW

**Comment by ricardopereira** [comment]
Testing the CocoaPod and it fails 😖

```
target 'CrashHistory' do
  use_frameworks!
  pod 'Ably', :path => '/Users/ricardopereira/Repositories/Whitesmith/ably-ios'
end
```

**Comment by ricardopereira** [comment]
CocoaPod and Carthage are working! 🙌

File: Source/ARTWebSocketTransport.m [hunk 1]
```diff
@@ -78,7 +78,12 @@ - (void)send:(NSData *)data withSource:(id)decodedObject {
         if ([decodedObject isKindOfClass:[ARTProtocolMessage class]]) {
             [_protocolMessagesLogger info:@"send %@", [decodedObject description]];
         }
+        #if COCOAPODS
+        [self.websocket send:data];
+        #else
+        // https://github.com/facebook/SocketRocket/issues/542
```
Comment (tcard) [inline Source/ARTWebSocketTransport.m]: Wow :(

**Comment by ricardopereira** [comment]
@tcard I think so but let them try the branch. Just for sure.

**Pull request merged**